nexus-sdk/src/main/java/io/nexusrpc/handler/OperationStartDetails.java

```diff
@@ -48,20 +51,31 @@ public String getRequestId() {
     return requestId;
   }
 
+  /**
+   * Links contain arbitrary caller information. Handlers may use these links as metadata on
+   * resources associated with and operation.
+   */
+  public List<Link> getLinks() {
+    return links;
+  }
+
   /** Builder for operation start details. */
   public static class Builder {
     private @Nullable String callbackUrl;
     private final Map<String, String> callbackHeaders;
     private @Nullable String requestId;
+    private List<Link> links;
```

Comment:
```suggestion
    private final List<Link> links;
```

(same for OperationStartResult)